swap_meet/vendor.py

```diff
@@ -1,2 +1,87 @@
 class Vendor:
-    pass
\ No newline at end of file
+    def __init__(self, inventory=None):
+        self.inventory = inventory or []
+    
+    def add(self, item):
+        self.inventory.append(item)
+        return item
+
+    def remove(self, item):
+        if item in self.inventory:
+            self.inventory.remove(item)
+            return item
+        
+        return False
+    
+    def get_by_id(self, item_id):
+        for item in self.inventory:
+            # we do not need to import id bc we're not inheriting anything
+            if item.id == item_id:
+                return item
+        return None
+    
+    def swap_items(self, other_vendor, my_item, their_item):
+        '''
+        other_vendor reps the friend that the vendor(self) is swapping with
+        my_item reps the item I am giving away
+        their_item reps item other_vendor plans to give
+        '''
+        if my_item not in self.inventory or their_item not in other_vendor.inventory:
+            return False
+
+        # remove my_item from inv
```

Comment:
[opinion] I think you can remove comments that repeat what the line of code below them does without adding any color.